mmv1/templates/terraform/examples/eventarc_workflows.tf.erb

```diff
@@ -0,0 +1,101 @@
+# [START eventarc_terraform_enableapis]
+# Used to retrieve project_number later
+data "google_project" "project" {
+  provider = google-beta
+}
+
+# Enable Eventarc API
+resource "google_project_service" "eventarc" {
+  provider = google-beta
+  service            = "eventarc.googleapis.com"
+  disable_on_destroy = false
+}
+
+# Enable Pub/Sub API
+resource "google_project_service" "pubsub" {
+  provider = google-beta
+  service            = "pubsub.googleapis.com"
+  disable_on_destroy = false
+}
+
+# Enable Workflows API
+resource "google_project_service" "workflows" {
+  provider = google-beta
+  service            = "workflows.googleapis.com"
+  disable_on_destroy = false
+}
+
+# [END eventarc_terraform_enableapis]
+
+# [START eventarc_workflows_create_serviceaccount]
+
+# Create a service account for Eventarc trigger and Workflows
+resource "google_service_account" "eventarc_workflows_service_account" {
+  provider = google-beta
+  account_id   = "eventarc-workflows-sa"
+  display_name = "Eventarc Workflows Service Account"
+}
+
+# Grant the logWriter role to the service account
+resource "google_project_iam_binding" "project_binding_eventarc" {
+  provider = google-beta
+  project = data.google_project.project.id
+  role    = "roles/logging.logWriter"
+
+members = ["serviceAccount:${google_service_account.eventarc_workflows_service_account.email}"]
+
+  depends_on = [google_service_account.eventarc_workflows_service_account]
+}
+
+# Grant the workflows.invoker role to the service account
+resource "google_project_iam_binding" "project_binding_workflows" {
+  provider = google-beta
+  project = data.google_project.project.id
+  role    = "roles/workflows.invoker"
+
+members = ["serviceAccount:${google_service_account.eventarc_workflows_service_account.email}"]
+
+  depends_on = [google_service_account.eventarc_workflows_service_account]
+}
+
+# [END eventarc_workflows_create_serviceaccount]
+
+# [START eventarc_workflows_deploy]
+# Define and deploy a workflow
+resource "google_workflows_workflow" "workflows_example" {
+  name            = "<%= ctx[:vars]['pubsub_workflow_tf'] %>"
+  provider = google-beta
+  region          = "us-central1"
+  description     = "A sample workflow"
+  service_account = google_service_account.eventarc_workflows_service_account.id
+  # Imported main workflow YAML file
+  source_contents = templatefile("text-fixtures/workflow.yaml",{})
```

Comment:
```suggestion
  source_contents = templatefile("test-fixtures/workflow.yaml",{})
```
sorry for the misunderstanding, I think you'll want `test` here, not `text` :)

Reply:
Oh gosh! I'm so sorry, I have corrected it now.

Reply:
no worries :) thanks!